python/tvm/auto_scheduler/cost_model/xgb_model.py

```diff
@@ -116,12 +117,17 @@ def __init__(self, verbose_eval=25, num_warmup_sample=100, seed=None):
         self.plan_size = 32
         self.num_warmup_sample = num_warmup_sample
         self.verbose_eval = verbose_eval
+        self.model_file = model_file
+        if model_file:
+            logger.info("XGBModel: Load pretrained model from %s...", model_file)
+            self.load(model_file)
```

Comment:
Remove this. No python/sklearn model has api or behavior like this.

Reply:
Removed.